src/cascadia/TerminalSettingsModel/Command.idl

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Terminal.Settings.Model
         ActionAndArgs Action;
         String KeyChordText;
 
-        String Icon;
+        String IconPath;

```

Comment:
but but but iuts' not a path
so sad about this :|

Reply:
I'd really really love for this to not be totally insane.

`IconSource`? idk. When I get more time, I'm gonna set up a simple repro for the WinUI team to tell me exactly wtf is going on here.